src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const resultArray = [];
+
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i].type) {
```

Comment:
for of loop will be better 
```suggestion
  for (const action of actions) {
    switch (actions.type) {
```